### DISCUSSION TITLE
Is Headless Chrome supposed to work on Demo site?

### DISCUSSION THREAD

**Discussion by NiklasBr** [pimcore/pimcore#12651] [General]
I'm trying to debug an issue locally with print containers and when trying the same thing on demo.pimcore.fun it appears stuck here:

![image](https://user-images.githubusercontent.com/279826/178952053-e371080e-0c20-4ed6-a1dc-3d5cc15ca7b7.png)
![image](https://user-images.githubusercontent.com/279826/178952087-bb96212d-d2b5-4d09-a097-825e753f7a36.png)

**Comment by kingjia90** [comment]
~By looking at this https://github.com/pimcore/pimcore/issues/10655#issuecomment-947378454 feels like might not properly work, but could be outdated.~
Everything works fine with WkHtmltoPdf (Test under Web-to-Print Settings and PDF Generation), only Headless Chrome is not working, maybe Spiritix is not properly installed
https://github.com/pimcore/pimcore/blob/c55ce3a1bd3327b6a46f3b6c75b056fa081f25a4/composer.json#L184

**Reply by NiklasBr** [reply]
Made a PR https://github.com/pimcore/demo/pull/328
